account/guides/setupNetworkBridge.md

```diff
@@ -23,6 +29,8 @@ In order to route the outbound calls through Bandwidth, you'll need to create a
 |------------|----------|-------------|
 | `Username` | required | String identifying the user. |
 | `Domain`   | optional | String refining the identity of the user. The Domain will be joined to the UserName with an @ to create a composite username. For example, the UserName bob could be combined with the domain somewhere.com to create a composite username bob@somewhere.com |
```

Comment:
Should probably change this to `String defining ...` if you update the other instance of this that Matt pointed out.

Reply:
done